tests/test_redact.py

```diff
@@ -22,12 +23,18 @@ def svs_input_path(data_dir, request) -> Path:
     return data_dir / "input" / request.param
 
 
+@freeze_time("2023-05-12 12:12:53")
+def test_create_redact_dir(tmp_path):
+    output_dir = redact.create_redact_dir(tmp_path)
```

Comment:
Perhaps you could make this a more challenging test case, by adding a non-existent additional level after `tmp_path`. If future changes remove `parents=True`, the test should fail.

Reply:
added in a0c2bf04704b088fb3ba1cfdeb17ff930b8983bf